Sources/MpcNET/Commands/Database/FindAndSearchCommands.cs

```diff
@@ -160,7 +441,36 @@ public BaseFilterCommand(ITag tag, string searchText, int windowStart = -1, int
         /// <param name="windowEnd">End of the portion of the results desired</param>
         public BaseFilterCommand(List<KeyValuePair<ITag, string>> filters, int windowStart = -1, int windowEnd = -1)
```

Comment:
> I think the List<KeyValuePair<ITag, string>> should be removed entirely from BaseFilterCommand, and all the constructors that use it should create a FilterTag object instead.

For example, here:  

```suggestion
        public BaseFilterCommand(List<KeyValuePair<ITag, string>> filters, int windowStart = -1, int windowEnd = -1, FilterOperator operand = FilterOperator.Equal)
        {
        completeFilters = new List<IFilter>();
        completeFilters.AddRange(filters.Select(kvp => new FilterTag(kvp.Key, Kvp.Value, operand )).ToList());
        
        _start = windowStart;
        _end = windowEnd;
        }
```  

The operator can then be set in each FilterCommand constructor (and we can remove the old `Operand` static strings laying around! 🎊)